src/styles/Resizer.scss

```diff
@@ -2,41 +2,41 @@
 
```

Comment:
i dont think any files are actually using the `Resizer` class?? it's in `EditorAndOutput.js` but no one ever specifies for the `Resizer` class?